cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -102,8 +102,9 @@
 
 default['travis_build_environment']['rebar_url'] = \
   'https://github.com/rebar/rebar/wiki/rebar'
-default['travis_build_environment']['rebar3_url'] = \
-  'https://s3.amazonaws.com/rebar3/rebar3'
+default['travis_build_environment']['rebar3_url'] = 'https://s3.amazonaws.com/rebar3/rebar3'
```

Comment:
How come this is being downloaded from amazon s3 rather than github? 

https://github.com/erlang/rebar3/releases

https://github.com/erlang/rebar3/releases/download/3.6.1/rebar3

Is it because the travis downloader (don't know what it is) doesn't support redirects ?

```
curl https://github.com/erlang/rebar3/releases/download/3.6.1/rebar3
```

```
< HTTP/1.1 302 Found
< Server: GitHub.com
< Date: Wed, 05 Sep 2018 21:43:37 GMT
< Content-Type: text/html; charset=utf-8
< Transfer-Encoding: chunked
< Status: 302 Found
< Cache-Control: no-cache
< Vary: X-PJAX
< Location: https://github-production-release-asset-2e65be.s3.amazonaws.com/26610130/5ca53498-7888-11e8-9f80-3b401b386fb7?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIW
NJYAX4CSVEH53A%2F20180905%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20180905T214337Z&X-Amz-Expires=300&X-Amz-Signature=387a8a1740d2402154a0fbb3821d488d2f100edccb84f48f86765329dd
f2fd2e&X-Amz-SignedHeaders=host&actor_id=0&response-content-disposition=attachment%3B%20filename%3Drebar3&response-content-type=application%2Foctet-stream
  
````